docs/index.rst

```diff
@@ -91,35 +91,37 @@ requirements. This is the “batteries-included” solution probably suitable fo
 
     python example_peninsula.py --fieldset 100 100
 
-4. Optionally, if you want to run all the examples and tutorials, you may want to download the example hydrodynamic fields by downloading the `pull_data.py <https://raw.githubusercontent.com/OceanParcels/parcels/master/parcels/scripts/pull_data.py>`_ file and then run::
+4. Optionally, if you want to run all the examples and tutorials, you can get tutorials and example data by running::
 
-    python pull_data.py
+    parcels_get_examples $HOME/parcels_first_steps
```

Comment:
Assuming users are doing all this from their $HOME directory? I reckon that most users would actually first create a different directory before they install Parcels, from which they run the commands? Can't we simply say?
```
parcels_get_examples parcels_first_steps
```
And I'm not entirely sure about the `first_steps` thing; sounds a bit too much like it's a baby-toy?? ;-)

Reply:
Yes, this an be simplified. <https://github.com/OceanParcels/parcels/pull/235/files#diff-ff3788e5426d53da20e4414d0ad3dbbdR94> will also work with a relative path.

And maybe let's change `parcels_first_steps` to `parcels_examples`?

Reply:
This still needs to change to `parcels_get_examples examples/` in the documentation too?